src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,18 @@
+const CHECK_STATUS = 'categories/CHECK_STATUS';
+
+const INITIAL_STATE = [];
+
+export const checkStatus = () => ({
+  type: CHECK_STATUS,
+});
+
+const categoriesReducer = (state = INITIAL_STATE, action) => {
+  switch (action.type) {
+    case CHECK_STATUS:
+      return 'Under construction';
+    default:
+      return state;
+  }
+};
```

Comment:
- [ ] Same here, Please use `createSlice` from `redux-toolkit` to create a new redux slice(reducer and action).

[Hint] It's supposed to be like that:

```javascript

import { createSlice } from "@reduxjs/toolkit"

export const categoriesSlice = createSlice({
  name: "categories",
  initialState: {
    categories: [],
  },
  reducers: {
    checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    },
  },
})

export const { checkStatus } = categoriesSlice.actions

export default categoriesSlice.reducer
```